src/App.js

```diff
@@ -1,10 +1,14 @@
 import "./App.css";
+import Button from "./components/Button";
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
         <h1>Volunteer App</h1>
+        <Button>Register</Button>
```

Comment:
We shouldn't be adding these into `App` file. 
Could you created a new folder called `modules` inside src. 
Then create a file inside modules called `home.js`
And inside that we will render `<Button>Register</Button>`
And here we can then render `<Home/>`

Reply:
Also just a thought, lets get react router into this project early and not render `Home` here but render the route `/`